docker-images/Dockerfile

```diff
@@ -0,0 +1,23 @@
+ARG PYTHON_VERSION=3.11.7
+FROM python:${PYTHON_VERSION}-slim as base
+
+# Prevents Python from writing pyc files.
+ENV PYTHONDONTWRITEBYTECODE=1
+
+# Keeps Python from buffering stdout and stderr to avoid situations where
+# the application crashes without emitting any logs due to buffering.
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR .. .
+
+# Download dependencies from pyproject.toml
+#RUN pip install
+
+# Copy the source code into the container.
+COPY .. .
```

Comment:
at this point we need to:
- copy poetry files (pyproject and lock)
- copy project files (only the package itself)

then we need to tell poetry not to create virtual environments and install project dependencies (like in github actions)